Also allow Laravel 6 installation

danielbachhuber opened the pull request.

danielbachhuber committed:
Also allow Laravel 6 installation

danielbachhuber commented:
Might not be so easy. I'm running into a cryptic error with this change:

```
In Container.php line 806:

  Target class [Loonpwn\Swiftype\Console\Commands\PurgeAllDocuments] does not exist.

In Container.php line 804:

  Class Loonpwn\Swiftype\Console\Commands\PurgeAllDocuments does not exist
```

loonpwn commented:
Hey @danielbachhuber 

Thought I recognised that profile photo, fan of your WP-CLI project!

I have just pushed out a version 1.0.0 release which should be Laravel 6 compatible, I also changed the library to extend off of the official App search package released by Swiftype.

I need to do some proper testing on it still but you can try that version if this is something that you still need and weren't able to resolve yourself yet.

danielbachhuber commented:
> Thought I recognised that profile photo, fan of your WP-CLI project!

Hey, thanks!

> I have just pushed out a version 1.0.0 release which should be Laravel 6 compatible, I also changed the library to extend off of the official App search package released by Swiftype.

Ok, neat :)

> I need to do some proper testing on it still but you can try that version if this is something that you still need and weren't able to resolve yourself yet.

I actually went a slightly different route. I took some of the code in this library and implemented it as a Scout Engine. Have you considered that?
